### PR TITLE
Let's do it! 💃

### PR DESCRIPTION
**Pull request by dougalg**
Probably needs tests or something

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+const program = require('commander');
+const pkg = require('../package.json');
+const { FORMAT, getExt } = require('./formatters');
+const StylegudParser = require('./stylegud-parser');
+
+
+const formatNames = Object.keys(FORMAT).map((k) => FORMAT[k]);
```
Comment (jonmelcher) [inline src/cli.js]: This should work here?  https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_objects/Object/values
`const formatNames = Object.values(FORMAT);`

File: src/formatters.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+     * @param {String} [options.export=sections] The variable name to export from the JS module
+     * @returns {String} A string of a JS file with es module export synax
+     */
+    format(json, options) {
+        const exportName = options.export || 'sections';
+        return `export const ${exportName} = ${JSON.stringify(json)};`;
+    }
+};
+
+const CommonJSFormater = {
```
Comment (jonmelcher) [inline src/formatters.js]: typo here: `CommonJSFormatter`

File: src/cli.js (hunk 1 above)
Comment (dougalg) [inline src/cli.js]: Not supported yet in Node :(

File: src/formatters.js (hunk 2 above)
Comment (dougalg) [inline src/formatters.js]: thanks!

**Pull request merged**